jina/executors/indexers/keyvalue.py

```diff
@@ -55,6 +55,8 @@ def __init__(self, *args, **kwargs):
         self._page_size = mmap.ALLOCATIONGRANULARITY
 
     def add(self, keys: Iterator[int], values: Iterator[bytes], *args, **kwargs):
+        if len(keys) != len(values):
```

Comment:
dont think iterator has `__length__` (telling from from the type hint)

Reply:
ah yes, thanks